### DISCUSSION TITLE
Use resources inside other resources

### DISCUSSION THREAD

**Discussion by thebiltheory** [mockapi-io/docs#11] [General]
Consider these 2 endpoints

- Customers
- Quotes

Is it possible inside a quote to reference the customers resource? 

```
{
  id: '1',
  ...
  customer: # comes from the customers resource
...
}
```

**Comment by nsmirn0v** [comment]
If there is a relationship defined between these two resources (i.e. customers has-many quotes) then it's is possible:

**Include parent resource**
https://github.com/mockapi-io/docs/assets/1930058/3a9f89a8-8044-4c84-8ed8-3e00acb3622c

**Include child resource**
https://github.com/mockapi-io/docs/assets/1930058/9c5e9a09-b0f8-4893-8be2-724e976383df
